main.go

```diff
@@ -13,15 +13,27 @@ import (
 
 var collection string
 var minCheckins int
+var is2010 bool
+var tfs2010Collections = [18]string{"CareSystems", "ClaimsEncounters", "CoreSystems","CorpSystemsNon-Erp", "CRM", "DBA","DefaultCollection","Encounters", "E-portal", "ITOps","Octo","ODSDWBI",
```

Comment:
Wrap the long lines. Don't make readers have to scroll right to see what is going on.